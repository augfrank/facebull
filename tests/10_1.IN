M0   C0   C1  2151
M1   C0   C3  2630
M2   C1   C5  2498
M3   C1   C8  2019
M4   C2   C7  2973
M5   C3   C0  2754
M6   C3   C1  2036
M7   C4   C8  2681
M8   C4   C9  2627
M9   C5   C4  2390
M10   C6   C2  2177
M11   C6   C4  2697
M12   C6   C5  2413
M13   C7   C5  2286
M14   C8   C6  2198
M15   C9   C0  2545
M16   C9   C4  2878
M17   C9   C5  2224
