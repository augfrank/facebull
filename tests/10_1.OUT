23881
1 3 4 6 8 9 10 13 14 15
